Makefile

```diff
@@ -100,12 +103,20 @@ all: test docker-build
 # Run tests
 test: build test-only
 
-test-only:
+LOCALBIN ?= $(shell pwd)/hack
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+
+test-only: envtest
 	@echo
 	@echo "If tests fail due to no matches for kind \"CustomResourceDefinition\" in version \"apiextensions.k8s.io/v1\","
 	@echo "please remove the old kubebuilder and reinstall it - https://book.kubebuilder.io/quick-start.html#installation"
 	@echo
```

Comment:
Can these lines be removed now? I think the makefile now does what this is telling the user to do?

Reply:
@adrianludwin 
> Can these lines be removed now? 

I think so.  
Now, `kubebuilder` is installed as single binary like [this(`/usr/local/bin/kubebuilder`)](https://kubebuilder.io/quick-start.html#installation) not like [previously](https://github.com/kubernetes-sigs/controller-runtime/issues/90#issuecomment-494878527).  
So these lines may be useless.